src/Silk.Core.Logic/Commands/General/ServersCommand.cs

```diff
@@ -18,7 +19,7 @@ public async Task Servers(CommandContext ctx)
         }
         private static int GetGuildCount()
         {
-            return Discord.Bot.Instance!.Client.ShardClients.Values.SelectMany(s => s.Guilds.Keys).Count();
+            return Main.ShardClient.ShardClients.Values.SelectMany(s => s.Guilds.Keys).Count();
```

Comment:
Again, use Sum instead of SelectMany. Don't produce an IEnumerable if you don't need one. Compiler may optimize this away but I don't know and it smells a little funny to me.